openfisca_france/param/param.xml

```diff
@@ -2781,7 +2784,16 @@
         <VALUE deb="2010-01-01" fin="2011-12-31" valeur="878" />
         <VALUE deb="2012-01-01" fin="2012-12-31" valeur="960" />
         <VALUE deb="2013-01-01" fin="2013-12-31" valeur="1016" />
-        <VALUE deb="2014-01-01" fin="2014-12-31" valeur="1021" />
```

Comment:
Reforms where not in good shape and it was difficult to modify the
parameters. So we changed it in the code and the trick was done

On Tue, Sep 22, 2015 at 5:57 PM adrienpacifico <notifications@github.com>
wrote:

> In openfisca_france/param/param.xml
> <https://github.com/openfisca/openfisca-france/pull/324#discussion_r40104670>
> :
>
> > @@ -2781,7 +2784,16 @@
> >          <VALUE deb="2010-01-01" fin="2011-12-31" valeur="878" />
> >          <VALUE deb="2012-01-01" fin="2012-12-31" valeur="960" />
> >          <VALUE deb="2013-01-01" fin="2013-12-31" valeur="1016" />
> > -        <VALUE deb="2014-01-01" fin="2014-12-31" valeur="1021" />
>
> It's been added by @clbe <https://github.com/clbe> on 2014/10/30, I don't
> remember but probably before the decote reform and suppression of the first
> tax slice has been announced.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openfisca/openfisca-france/pull/324/files#r40104670>.
>